cirq/ops/three_qubit_gates_test.py

```diff
@@ -201,6 +201,7 @@ def test_identity_multiplication():
         ),
     ],
 )
+@cirq.testing.skip_if_module_not_exists(module="cirq.google")
```

Comment:
This is where it gets a bit ugly and would be great to spend some time on the proper solution rather than trying to squeeze it into this PR - so it will be probably another PR.

These tests have are a bit troublesome in a sense that it creates coverage for both the cirq.google stuff (ConvertToXmonGates(), Foxtail validation) functionality as well as the three qubit gates themselves (e.g. their decomposition logic). In order to fix this we should separate the concerns and create nicely isolated tests for both of these sides - cirq.google things should be validated against the three qubit gates as well on the cirq.google side and then we can have the specific functionality tested in isolation in the three qubit gates side...ugh.

Reply:
Hmmm. It sounds like this would break the `cirq-only` coverage checks I recommended in another comment, which raises serious warning flags for me. I don't mind if those changes are a separate PR, but I would _highly_ prefer to see that PR lined up and ready to go before approving this one - especially if those changes are as non-trivial as it sounds.

Reply:
Yeah, agreed, I'll look into this issue more deeply before progressing with this PR.

Reply:
It was simpler than I thought - opened https://github.com/quantumlib/Cirq/pull/3928